modules/specialized-guides/pages/large-deployments/hub-online-sync.adoc

```diff
@@ -0,0 +1,213 @@
+. by direct registration from the hub, without any interactions with the peripheral server. 
+  The steps for this method are described in <<direct-registration>>.
+
+
+
+=== Registration from peripheral server by token generation
+
+Before being registered to the hub server, a token needs to be generated on the peripheral server and passed it to the administrator of the hub server.
+
+[[peripheral-token-generation]]
+.Procedure: Generating token on the peripheral server
+. On the peripheral server, go to menu:Admin[Hub Configuration > Access Tokens].
+. Click button btn:[Add token] and select option [literal]``Issue new token``.
+. In the field [literal]``Server FQDN`` on the form that opens type the FQDN of the hub server that will be using this token.
+. Click btn:[Issue].
+. A new form with the successfully generated token appears and button btn:[Copy].
+
++
+
+[IMPORTANT]
+==== 
+The only time token is displayed is at the time of its creation.
+====
+
+. Once generated, the token appears on the screen [literal]``Access Tokens``.
+
+The generated token needs to be transferred to the hub server before it can be used. 
+
+
+[[token-transfer]]
+.Procedure: Registering to the hub server with the token
+. On the hub server, go to menu:Hub Configuration[Peripherals Configuration].
+. Click button btn:[Add peripheral].
+  A new form [literal]``Register a new peripheral server`` opens.
+. In the field [literal]``Peripheral Server FQDN`` enter the name of the peripheral server.
+. In the field [literal]``Registration mode`` select option [literal]``Existing token``.
+. In the field [literal]``Token`` paste the token that was created on the peripheral server.
+. In the field [literal]``Root CA certificate`` specify the certificate using one of the options:
+  * Use option [literal]``Not needed`` if both hub and peripheral servers have the same certificate authority.
+  * Use option [literal]``Upload a file`` if the servers have different certificate authorities to upload a certificate file.
+  * Use option [literal]``Paste a PEM certificate`` to paste a certificate.  
+. Click button btn:[Register].
+  A newly registered peripheral serevr will appear on screen [literal]``Peripherals Configuration``.
+
+
+
+=== Registration from the hub server directly
+
+It is possible to initiate the registration of a peripheral server from hub server, without any interaction with the peripheral server.
+
+[[direct-registration]]
+.Procedure: Direct registering from the hub server
+
+. On the hub server, go to menu:Hub Configuration[Peripherals Configuration].
+. Click button btn:[Add peripheral].
+  A new form [literal]``Register a new peripheral server`` opens.
+. In the field [literal]``Peripheral Server FQDN`` enter the name of the peripheral server.
+. In the field [literal]``Registration mode`` select option [literal]``Administrator User/Password``.
+. In the fields [literal]``Username`` and [literal]``Password`` enter the credentials for the peripheral server.
```

Comment:
Please say, that these credentials must be from a "SUSE Manager Administrator" of the peripheral server.